### DISCUSSION TITLE
How did the github workflow job Summaries create a table

### DISCUSSION THREAD

**Discussion by mannicui** [community/community#37628] [Actions] [answered]
How did the github workflow job Summaries create a table  like this 

<img width="401" alt="job" src="https://user-images.githubusercontent.com/97588478/198576953-58a51d11-543c-40de-a11f-eb8317087a8a.png">

**Comment by airtower-luna** [comment, marked as answer]
The job summary is written as Markdown, so you can use any Markdown syntax supported by GitHub. The colorful symbols [are emoji](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#using-emoji) (`:name:` syntax). For how to create tables in Markdown, see [Organized data with tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables).
